lib/stdlib/src/shell_docs.erl

```diff
@@ -31,6 +31,8 @@
 %% Convinience functions
 -export([get_doc/1, get_doc/3, get_type_doc/3, get_callback_doc/3]).
 
+-export_type([chunk_elements/0, chunk_element_attr/0]).
```

Comment:
[It's used in a spec](https://github.com/erszcz/otp/blob/6e86552dffc1a5a2329958553374c1b6c83b0a42/lib/edoc/src/edoc_layout_chunks.erl#L383), since I wanted to highlight that the HTML tree produced by the layout chunk is later consumed by `shell_docs`. However, it's not critical, this type does not have to be exported - just let me know and I'll revert that.